/groups/behroozi/hbowden/cocoa/Cocoa/installation_scripts/create_link_all_projects.sh: line 55: /dev/tty: No such device or address
Traceback (most recent call last):
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/calc_dvs.py", line 21, in <module>
Traceback (most recent call last):
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/calc_dvs.py", line 21, in <module>
    config = Config(configfile)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 36, in __init__
Traceback (most recent call last):
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/calc_dvs.py", line 21, in <module>
Traceback (most recent call last):
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/calc_dvs.py", line 21, in <module>
    config = Config(configfile)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 36, in __init__
Traceback (most recent call last):
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/calc_dvs.py", line 21, in <module>
Traceback (most recent call last):
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/calc_dvs.py", line 21, in <module>
    config = Config(configfile)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 36, in __init__
    config = Config(configfile)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 36, in __init__
Traceback (most recent call last):
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/calc_dvs.py", line 21, in <module>
Traceback (most recent call last):
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/calc_dvs.py", line 21, in <module>
    config = Config(configfile)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 36, in __init__
    config = Config(configfile)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 36, in __init__
    config = Config(configfile)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 36, in __init__
    config = Config(configfile)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 36, in __init__
Traceback (most recent call last):
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/calc_dvs.py", line 21, in <module>
    config = Config(configfile)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 36, in __init__
Traceback (most recent call last):
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/calc_dvs.py", line 21, in <module>
    config = Config(configfile)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 36, in __init__
    self.load_lkl(config_args_lkl)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 52, in load_lkl
    self.load_lkl(config_args_lkl)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 52, in load_lkl
    self.load_lkl(config_args_lkl)
    self.load_lkl(config_args_lkl)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 52, in load_lkl
    self.load_lkl(config_args_lkl)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 52, in load_lkl
    self.load_lkl(config_args_lkl)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 52, in load_lkl
    self.load_lkl(config_args_lkl)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 52, in load_lkl
    self.load_lkl(config_args_lkl)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 52, in load_lkl
    self.load_lkl(config_args_lkl)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 52, in load_lkl
    self.load_lkl(config_args_lkl)
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 52, in load_lkl
  File "/groups/behroozi/hbowden/cocoa/Cocoa/projects/roman_real/cocoa_emu/config.py", line 52, in load_lkl
    dataset = readDatasetFile(self.config_args_lkl['data_file'], 
KeyError: 'data_file'
    dataset = readDatasetFile(self.config_args_lkl['data_file'], 
KeyError: 'data_file'
    dataset = readDatasetFile(self.config_args_lkl['data_file'], 
KeyError: 'data_file'
    dataset = readDatasetFile(self.config_args_lkl['data_file'], 
KeyError: 'data_file'
    dataset = readDatasetFile(self.config_args_lkl['data_file'], 
KeyError: 'data_file'
    dataset = readDatasetFile(self.config_args_lkl['data_file'], 
KeyError: 'data_file'
    dataset = readDatasetFile(self.config_args_lkl['data_file'], 
KeyError: 'data_file'
    dataset = readDatasetFile(self.config_args_lkl['data_file'], 
KeyError: 'data_file'
    dataset = readDatasetFile(self.config_args_lkl['data_file'], 
KeyError: 'data_file'
    dataset = readDatasetFile(self.config_args_lkl['data_file'], 
KeyError: 'data_file'
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[22177,1],7]
  Exit code:    1
--------------------------------------------------------------------------
